en/docs/develop/localization-support-in-identity-server.md

```diff
@@ -65,4 +65,10 @@ Similarly, you can enable localization for accountrecoveryendpoint by
 applying the same above steps 1 through 7 in the previous section
 starting with the following location.
 
-`<IS_HOME>/repository/deployment/server/webapps/accountrecoveryendpoint/WEB-INF/classes/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/`
\ No newline at end of file
+`<IS_HOME>/repository/deployment/server/webapps/accountrecoveryendpoint/WEB-INF/classes/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/`
+
+### Configuring Localization for My Account and Console
+
+We maintain the i18n (localization) resources for My Account and the Management Console in a separate module inside the 
```

Comment:
@deshankoswatte - Can you check my edits here? When we say `Console`, we mean the `Management Console` I presume.